pkgs/build-support/setup-hooks/multiple-outputs.sh

```diff
@@ -192,9 +193,9 @@ _multioutPropagateDev() {
         propagaterOutput="$outputFirst"
     fi
 
-    # Default value: propagate binaries, includes and libraries
+    # Default value: propagate includes and libraries
     if [ -z "${propagatedBuildOutputs+1}" ]; then
-        local po_dirty="$outputBin $outputInclude $outputLib"
+        local po_dirty="$outputInclude $outputLib"
```

Comment:
No idea how much this is going to break, but we shouldn't really have been doing that anyway. If mkDerivation doesn't reset nativeBuildInputs to select `.dev` this might even work as is

Reply:
Reverted in favour of https://github.com/NixOS/nixpkgs/pull/318226